test/alias.test.ts

```diff
@@ -0,0 +1,144 @@
+import { getAlias, isAlias } from '../src/alias';
+import { Buffer } from 'buffer/';
+
+test('isAlias is reflexive', () => {
+    const address = new AccountAddress(
+        bs58check.encode(
+            Buffer.from(
+                '01e718721402249e81f8fedcba6027f1c9bcb4445e9433b7905d579d83ec000002',
+                'hex'
+            )
+        )
+    );
+    expect(isAlias(address, address)).toBeTruthy();
+});
+
+test('isAlias: Addresses with first 29 bytes in common are aliases', () => {
+    const address = new AccountAddress(
+        bs58check.encode(
+            Buffer.from(
+                '01e718721402249e81f8fedcba6027f1c9bcb4445e9433b7905d579d83ec000002',
+                'hex'
+            )
+        )
+    );
+    const alias = new AccountAddress(
+        bs58check.encode(
+            Buffer.from(
+                '01e718721402249e81f8fedcba6027f1c9bcb4445e9433b7905d579d83ececb467',
+                'hex'
+            )
+        )
+    );
+    expect(isAlias(address, alias)).toBeTruthy();
+});
+
+test('isAlias: Addresses with differences in the 5th byte are not aliases', () => {
+    const address = new AccountAddress(
+        bs58check.encode(
+            Buffer.from(
+                '01e718721412249e81f8fedcba6027f1c9bcb4445e9433b7905d579d83ec000002',
+                'hex'
+            )
+        )
+    );
+    const alias = new AccountAddress(
+        bs58check.encode(
+            Buffer.from(
+                '01e718721402249e81f8fedcba6027f1c9bcb4445e9433b7905d579d83ec000002',
+                'hex'
+            )
+        )
+    );
+    expect(isAlias(address, alias)).toBeFalsy();
+});
+
+test('isAlias: Addresses with differences in the 29th byte are not aliases', () => {
+    const address = new AccountAddress(
+        bs58check.encode(
+            Buffer.from(
+                '01e718721402249e81f8fedcba6027f1c9bcb4445e9433b7905d579d83ececb467',
+                'hex'
+            )
+        )
+    );
+    const alias = new AccountAddress(
+        bs58check.encode(
+            Buffer.from(
+                '01e718721402249e81f8fedcba6027f1c9bcb4445e9433b7905d579d83e1ecb467',
+                'hex'
+            )
+        )
+    );
+    expect(isAlias(address, alias)).toBeFalsy();
+});
+
+test('getAlias: getAlias returns an alias', () => {
+    const address = new AccountAddress(
+        '4hXCdgNTxgM7LNm8nFJEfjDhEcyjjqQnPSRyBS9QgmHKQVxKRf'
+    );
+    const alias = getAlias(address, 1);
+    expect(isAlias(address, alias)).toBeTruthy();
+});
+
+test('getAlias: changing counter makes getAlias return different aliases', () => {
+    const address = new AccountAddress(
+        '4hXCdgNTxgM7LNm8nFJEfjDhEcyjjqQnPSRyBS9QgmHKQVxKRf'
+    );
+    const alias = getAlias(address, 1);
+    const otherAlias = getAlias(address, 100);
+    expect(otherAlias.address).not.toBe(alias.address);
+    expect(isAlias(otherAlias, alias)).toBeTruthy();
+});
+
+test('getAlias: last 3 bytes of alias matches counter', () => {
```

Comment:
I think this test with the minimum and maximum counter values would be good to have too.

Reply:
Added them to that test